Source/Core/Core/Core.cpp

```diff
@@ -112,16 +105,7 @@ struct HostJob
 static std::mutex s_host_jobs_lock;
 static std::queue<HostJob> s_host_jobs_queue;
 
-#ifdef ThreadLocalStorage
-static ThreadLocalStorage bool tls_is_cpu_thread = false;
-#else
-static pthread_key_t s_tls_is_cpu_key;
-static pthread_once_t s_cpu_key_is_init = PTHREAD_ONCE_INIT;
-static void InitIsCPUKey()
-{
-  pthread_key_create(&s_tls_is_cpu_key, nullptr);
-}
-#endif
+static thread_local bool tls_is_cpu_thread = false;
```

Comment:
Shouldn't an anonymous namespace be used instead of `static`? Feel free to ignore this comment since this file is full of `static` for internal linkage.

Reply:
Ideally, yes. However the variable was already static to begin with. This just removes the ThreadLocalStorage define